File: Programmers/Lv0/정사각형으로만들기.cpp
#include <string>
#include <vector>

using namespace std;

vector<vector<int>> solution(vector<vector<int>> arrs) {
    int rowCnt = arrs.size(), colCnt = arrs[0].size();
    if (rowCnt > colCnt) {
        int diff = rowCnt - colCnt;
        for (vector<int> &arr : arrs)
            for (int i = 0; i < diff; i++)
                arr.push_back(0);
    }
    else if (rowCnt < colCnt) {
        int diff = colCnt - rowCnt;
        for (int j = 0; j < diff; j++)
            arrs.push_back(vector<int>(colCnt));
    }
    return arrs;
}

File: Programmers/Lv0/정사각형으로만들기.py
def solution(arrs):
    rowCnt = len(arrs) 
    colCnt = len(arrs[0])
    if rowCnt > colCnt:
        diff = rowCnt - colCnt
        for arr in arrs:
            for _ in range(diff):
                arr.append(0)
    elif rowCnt < colCnt:
        diff = colCnt - rowCnt
        for _ in range(diff):
            arrs.append([0] * colCnt)
    return arrs